packages/oranger/oranger.2.0.3/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+authors: ["Francois Berenger"]
+homepage: "https://github.com/UnixJunkie/oranger"
+bug-reports: "https://github.com/UnixJunkie/oranger/issues"
+dev-repo: "git+https://github.com/UnixJunkie/oranger.git"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["tar" "xzf" "0.9.11.tar.gz"]
+  [conf-cmake:cmd "-S" "ranger-0.9.11/cpp_version" "-B" "ranger-build"]
+  [make "-C" "ranger-build"]
+]
+install: [
+  ["cp" "ranger-build/ranger" "%{bin}%/ml_rf_ranger"]
+]
+depends: [
+  "dune" {>= "1.0.0"}
+  "conf-cmake" {build}
+  "conf-wget" {build}
```

Comment:
```suggestion
  "conf-wget" {dev}
```
Is this only needed in the dev repository?

Reply:
I don't think, because the opam file mentions a script called install_ranger.sh.
This script dowloads the ranger tarball in case it is not already here.
So, each new install / build of oranger needs wget.

Reply:
That's going to run afoul of opam sandboxing.  You should specify any additional sources in `extra-sources` (with checksums) rather than fetching them from within the opam install script.

Reply:
> I don't think, because the opam file mentions a script called install_ranger.sh.
> This script dowloads the ranger tarball in case it is not already here.
> So, each new install / build of oranger needs wget.

That's not true, opam always get the tarball and will fail anyway if it cannot be downloaded, before it can reach the build phase.
I've made the necessary adjustments anyway so the script is not required

Reply:
conf-wget is not required anymore if opam can download the tarball